features/manage-position/ManagePosition.tsx

```diff
@@ -42,6 +42,7 @@ const Manager = () => {
       {method === "deposit" && <Deposit />}
       {method === "withdraw" && <Withdraw />}
       {method === "redeem" && <Redeem />}
+      {method === "transfer" && <FalseDoor />}
```

Comment:
I think we should just remove this option. Here and in `ManagePosition/MethodSelector`

Reply:
We do actually want it though, no? I've created an issue to track this: #63

Reply:
Yes eventually but for now why not just remove it?